### PR TITLE
GH-994 - Updating color for global header menu

### PR DESCRIPTION
**Pull request by asaadmahmood**
#### Summary
GH-994 - Updating color for global header menu

#### Ticket Link
https://github.com/mattermost/focalboard/issues/994

#### Screenshot
Screenshot 2021-08-16 at 11 38 14 PM

### CODE REVIEW

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:    @asaadmahmood , I think your editor has different formatting settings from the other devs. We should all get in sync other we'll keep clobbering each other's formatting.

**Review by hahmadia — APPROVED** [review]

File: mattermost-plugin/webapp/src/plugin.scss [hunk 1]
```diff
@@ -5,6 +5,24 @@
...
 }
 
-.focalboard-body .feature-global-header > header {
+.focalboard-body .feature-global-header>header {
     z-index: 1000;
-}
\ No newline at end of file
+}
+
+.feature-global-header {
```
Comment (jespino) [inline mattermost-plugin/webapp/src/plugin.scss:12]: This is going to affect the `.feature-global-header` of outside focalboard too. if you combine this with the previous one you can make it better. Something like:

```css
.focalboard-body .feature-global-header {
    >header {
        z-index: 1000;
    }
    .Menu {
...
```

**Review by jespino — CHANGES_REQUESTED** [review]
Small change needed.

File: mattermost-plugin/webapp/src/plugin.scss (hunk 1 above)
Comment (asaadmahmood) [reply mattermost-plugin/webapp/src/plugin.scss]: @jespino Done.

**Review by jespino — APPROVED** [review]

**Comment by jespino** [comment]
This can be tested after merge.

**Pull request merged**